OpenRA.Mods.Common/Graphics/ContrailRenderable.cs

```diff
@@ -75,7 +75,10 @@ public void Render(WorldRenderer wr)
 				var nextColor = Exts.ColorLerp(i * 1f / (length - 4), color, Color.Transparent);
 
 				if (!world.FogObscures(curPos) && !world.FogObscures(nextPos))
-					wcr.DrawLine(wr.Screen3DPosition(curPos), wr.Screen3DPosition(nextPos), screenWidth, curColor, nextColor);
+					if (curColor != nextColor)
+						wcr.DrawLine(wr.Screen3DPosition(curPos), wr.Screen3DPosition(nextPos), screenWidth, curColor, nextColor);
+					else
```

Comment:
Considering `var nextColor` is lerping towards `Color.Transparent`, how common is `curColor == nextColor`?